.github/workflows/check_query_identifiers.yaml

```diff
@@ -0,0 +1,45 @@
+name: check_query_identifiers
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+    types: [opened, reopened, synchronize]
+
+jobs:
+  format_check:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        python-version:
+          - "3.9"
+
+    runs-on: ${{ matrix.os }}
+
+    name: Run Check Query Identifiers
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # - name: Set up Python ${{ matrix.python-version }}
+      #   uses: actions/setup-python@v4
+      #   with:
+      #     python-version: ${{ matrix.python-version }}
+
+      # - name: Install dependencies
+      #   run: |
+      #     python -m pip install --upgrade uv
+      #     uv venv
+      #     uv pip install -r requirements.txt
+
+      # - name: Activate virtualenv
+      #   run: |
+      #     . .venv/bin/activate
+      #     echo PATH=$PATH >> $GITHUB_ENV
+
+      # - name: Run Python script
+      #   run: python src/scribe_data/check/check_query_identifiers.py
```

Comment:
Hello @catreedle and @andrewtavis, do you know why this was commented out? `check_query_identifiers.py` is done and I need to be sure all the commands are useful.

Reply:
I wanna know why, in the steps, we are setting up Python, installing dependencies, and activating a virtual environment but `.github/workflows/check_project_structure.yaml` does not have this style? is there a reason why?

Reply:
This is commented out so we don't get errors before everything is hooked up :)

Reply:
Maybe we don't need the environment for this one and it'd make sense to simplify the steps 😊

Reply:
Thanks @andrewtavis for the response!
yeah, maybe we don't need it. I can raise a PR and we quickly evaluate it :)

Reply:
Thanks, @DeleMike!